sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -21,6 +24,10 @@
 import com.azure.cosmos.models.CosmosClientTelemetryConfig;
 import com.azure.cosmos.models.CosmosPermissionProperties;
 
+// TODO: Use slf4j Logger and LoggerFactory instead of the core ClientLogger?
+// import org.slf4j.Logger;
+// import org.slf4j.LoggerFactory;
+
```

Comment:
ClientLogger (a core API class) is being used in the SDK ... but so is log4j factory/logger.
I am not sure what would be the better option - seems like one should be using ClientLogger.
I would remove this commented out imports or uncomment and remove the ClientLogger once it is clear what to use.